pkg/solana/chainwriter/transform_registry.go

```diff
@@ -7,19 +7,17 @@ import (
 
 	"github.com/gagliardetto/solana-go"
 	"github.com/mitchellh/mapstructure"
-	"github.com/smartcontractkit/chainlink-ccip/chains/solana/gobindings/ccip_offramp"
 	"github.com/smartcontractkit/chainlink-ccip/pkg/types/ccipocr3"
 )
 
 type ReportPostTransform struct {
-	ReportContext  [2][32]byte
-	Report         []byte
-	Info           ccipocr3.ExecuteReportInfo
-	AbstractReport ccip_offramp.ExecutionReportSingleChain
```

Comment:
Removed the AbstractReport field here since that field isn't in the off-chain or on-chain report objects, not sure how it got in here in the first place